### PR TITLE
contact us mail create and send

### PR DESCRIPTION
**Pull request by ghost**
first PR for contact us mail

### CODE REVIEW

**Review by Bruno-Fernandes-37 — DISMISSED** [review]

File: src/Controller/ContactController.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+use Symfony\Component\Mailer\MailerInterface;
+
+/**
+ * @Route("/contact", name="contact")
+ */
+
+class ContactController extends AbstractController
+{
+
+    public const MAILER = 'swap.wild@gmail.com';
```
Comment (RomainClair) [inline src/Controller/ContactController.php]: This should probably be a configuration environnement variable.
See : https://symfony.com/doc/current/configuration.html

**Review by RomainClair — CHANGES_REQUESTED** [review]
There is no validation of the form input.
You probably should use a dataclass behind the form and insert constraints annotations in it.

And you probably should add a dependency to have a Mailer service

**Review by Bruno-Fernandes-37 — DISMISSED** [review]

**Review by CECDO — DISMISSED** [review]
ok

**Review by Bruno-Fernandes-37 — APPROVED** [review]

**Review by CECDO — APPROVED** [review]

**Review by sebastien-violante — APPROVED** [review]

File: config/services.yaml [hunk 2]
```diff
@@ -4,7 +4,8 @@
 # Put parameters here that don't need to change on each machine where the app is deployed
 # https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
-
+    mailer_from: '%env(resolve:MAILER_FROM_ADDRESS)%'
```
Comment (RomainClair) [inline config/services.yaml:7]: You probably should modify the installation doc (or at least the README) to tell that a new environnement variable is needed

**Review by RomainClair — APPROVED** [review]

**Pull request merged**